app/src/util/tools.ts

```diff
@@ -358,6 +358,14 @@ export const getCTokenForToken = (token: string): string => {
   }
 }
 
+export const isCToken = (symbol: string): boolean => {
+  const tokenSymbol = symbol.toLowerCase()
+  if (tokenSymbol.startsWith('c')) {
+    return true
+  }
```

Comment:
This could potentially cause problems... I'm thinking we could probably call compound's API instead to do this check

Reply:
Oh yeah true! This has to be sync in some places, luckily I discovered we actually have a `CompoundTokenType` so I can check against that.